symfony/ldap/CVE-2018-11407.yaml

```diff
@@ -0,0 +1,26 @@
+title:     "CVE-2018-11407: Unauthorized access on a misconfigured LDAP server when using an empty password"
+link:      https://symfony.com/cve-2018-11407
+cve:       CVE-2018-11407
+branches:
+    2.8.x:
+        time:     2018-05-25 12:12:59
+        versions: ['>=2.8.0', '<2.8.41']
+    3.0.x:
+        time:     2018-05-25 12:44:29
+        versions: ['>=3.0.0', '<3.1.0']
+    3.1.x:
+        time:     2018-05-25 12:44:29
+        versions: ['>=3.1.0', '<3.2.0']
+    3.2.x:
+        time:     2018-05-25 12:44:29
+        versions: ['>=3.2.0', '<3.3.0']
+    3.3.x:
+        time:     2018-05-25 12:44:29
+        versions: ['>=3.3.0', '<3.3.17']
+    3.4.x:
+        time:     2018-05-25 13:17:43
+        versions: ['>=3.4.0', '<3.4.11']
+    4.0.x:
+        time:     2018-05-25 14:04:47
+        versions: ['>=4.0.0', '<4.0.11']
+reference: composer://symfony/ldap
```

Comment:
this looks wrong to me as the `LdapBindAuthenticationProvider` class is part of the `symfony/security-core` package

Reply:
CVE-2018-11407 must also be added to `symfony/security` then.